dgfit/dustmodel.py

```diff
@@ -607,6 +610,7 @@ def save_results(self, filename, OD, size_dist_uncs=[0]):
         cols = fits.ColDefs(all_cols_emis)
         tbhdu = fits.BinTableHDU.from_columns(cols)
         tbhdu.header.set("EXTNAME", "Emission", "emission MJy/sr/A(V)")
+        tbhdu.header.set("ISRF", ISRF, "The final ISRF strength")
```

Comment:
Maybe instead of "final" say "fitted" or even just leave it out.  Maybe just leave it out to allow for files from this code that are not fitted, but just computed.   Could be this kind of file is created without fitting for examples, etc.